esmvalcore/_task.py

```diff
@@ -593,6 +648,92 @@ def __str__(self):
         return txt
 
 
+def _get_response(url):
+    """Return information from CMIP6 Data Citation service in json format."""
+    json_data = False
+    if url.lower().startswith('https'):
+        try:
+            response = requests.get(url)
+            if response.status_code == 200:
+                json_data = response.json()
+            else:
+                logger.info('Error in the CMIP json link')
+        except IOError:
+            logger.info('Error in receiving the CMIP json file')
+    return json_data
+
+
+def _valid_json_data(data):
```

Comment:
Instead of validating, it might be a bit more robust to just use the `dict.get` method with a default option to retrieve the values from the json. That way you still get something meaningful even if a single field is missing (does this ever happen?)